src/Microsoft.Win32.Primitives/src/Microsoft.Win32.Primitives.csproj

```diff
@@ -8,7 +8,7 @@
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
     <IsPartialFacadeAssembly Condition="'$(TargetGroup)' == 'net463'">true</IsPartialFacadeAssembly>
     <!-- copy to uap10.1 until nuget makes the change to map that to ns1.7 -->
-    <PackageTargetFramework Condition="'$(TargetGroup)' == 'netstandard1.7'">netstandard1.7;uap10.1</PackageTargetFramework>
+    <PackageTargetFramework Condition="'$(TargetGroup)' == 'netstandard'">netstandard1.7;uap10.1</PackageTargetFramework>
```

Comment:
@ericstj should this property become `<PackageTargetFramework Condition="'$(TargetGroup)' == 'netstandard'">$(PackageTargetFramework);uap10.1</PackageTargetFramework>`?